Makefile

```diff
@@ -36,8 +37,7 @@ DOCKER_BUILD = docker build --no-cache \
 	    -C sonic \
 	    -f slave.mk \
 	    PLATFORM=$(PLATFORM) \
-	    DEBUG_BUILD=$(DEBUG_BUILD) \
```

Comment:
It is redundant. In case build contains local changes, "dirty" will be appended to version or used instead of git revision.